test/e2e/page-objects/flows/onboarding.flow.ts

```diff
@@ -20,14 +31,25 @@ export const createNewWalletOnboardingFlow = async (driver: Driver) => {
 
   const onboardingPasswordPage = new OnboardingPasswordPage(driver);
   await onboardingPasswordPage.check_pageIsLoaded();
-  await onboardingPasswordPage.createWalletPassword();
+  await onboardingPasswordPage.createWalletPassword(password);
 
   const secureWalletPage = new SecureWalletPage(driver);
   await secureWalletPage.check_pageIsLoaded();
   await secureWalletPage.revealAndConfirmSRP();
 };
 
-export const importSRPOnboardingFlow = async (driver: Driver) => {
+/**
+ * Import SRP onboarding flow
+ *
+ * @param driver - The WebDriver instance.
+ * @param seedPhrase - The seed phrase to import. Defaults to E2E_SRP.
+ * @param password - The password to use. Defaults to WALLET_PASSWORD.
+ */
+export const importSRPOnboardingFlow = async (
+  driver: Driver,
+  seedPhrase: string = E2E_SRP,
+  password: string = WALLET_PASSWORD,
+) => {
```

Comment:
nit: I think for case like this, where we have defaults, it is generally better to use an object as the function param. The reason for that is because, in the case we want to call it, but just passing some of the function arguments, it will be more clear. Example on calling it passing only driver and password, it would be like this:

`importSRPOnboardingFlow(driver, undefined, password);`

compared to using an object, it will be like this: 

`importSRPOnboardingFlow({ driver, password });` or `importSRPOnboardingFlow({ driver, password: MY_PWD });`

Another reason is, if the function grows, it will be easier and more flexible to manage params as an object too

Reply:
That's a good point, thanks ! Addressed in commit [a32c082](https://github.com/MetaMask/metamask-extension/pull/28202/commits/a32c082b50e9d9c9c7bd95fbd5862907d4d90d62)